DNN Platform/DotNetNuke.Web.Deprecated/UI/WebControls/DnnDatePicker.cs

```diff
@@ -33,11 +33,6 @@ public class DnnDatePicker : RadDatePicker
     {
         protected override void OnInit(EventArgs e)
         {
-			if (CultureInfo.CurrentCulture.Name == "ar-SA")
```

Comment:
I'll add this back, it was removed without my knowledge. Time gap between the start of this and finalizing the commit.